powerline/bindings/bar/powerline-bar.py

```diff
@@ -2,42 +2,45 @@
 # vim:fileencoding=utf-8:noet
 from __future__ import (unicode_literals, division, absolute_import, print_function)
 
+import os
 import sys
 import time
 
 from threading import Lock, Timer
 from argparse import ArgumentParser
 
-from powerline import Powerline
+from powerline.lemonbar import LemonbarPowerline
 from powerline.lib.encoding import get_unicode_writer
 
 
-class BarPowerline(Powerline):
-	get_encoding = staticmethod(lambda: 'utf-8')
-
-	def init(self):
-		super(BarPowerline, self).init(ext='wm', renderer_module='bar')
-
-
 if __name__ == '__main__':
-	parser = ArgumentParser(description='Powerline BAR bindings.')
+	parser = ArgumentParser(description='Powerline lemonbar bindings.')
 	parser.add_argument(
 		'--i3', action='store_true',
 		help='Subscribe for i3 events.'
 	)
+	parser.add_argument(
+		'--width', action='store', type=int,
+		default=None, help='Maximum width the bar can occupy'
+	)
+	parser.add_argument(
+		'theme', nargs='?', default='default',
+		help='The theme to render')
 	args = parser.parse_args()
-	powerline = BarPowerline()
+	powerline = LemonbarPowerline(theme=args.theme)
+	powerline.update_renderer()
+	powerline.logger.warn("the 'bar' bindings are deprecated, please switch to 'lemonbar'")
```

Comment:
Do not use `powerline.logger`, it is for internal use only. `powerline.pl` must be used instead.